tests/Frameworks/WordPress/Version_6_1/scripts/xdebug.ini

```diff
@@ -0,0 +1,7 @@
+zend_extension=xdebug-3.2.0RC1.so
+xdebug.mode=develop,debug
+xdebug.start_with_request=yes
+xdebug.log=/var/log/xdebug.log
```

Comment:
FYI, on startup I saw this in the logs:

```
version_6_1-app-1                | [Thu Jan 26 00:36:58 2023] Xdebug: [Log Files] File '/var/log/xdebug.log' could not be opened.
version_6_1-app-1                | [Thu Jan 26 00:36:58 2023] Xdebug: [Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).
```

Not sure if it matters, didn't try to debug anything.

Reply:
> `[Log Files] File '/var/log/xdebug.log' could not be opened.`
Fixed in https://github.com/DataDog/dd-trace-php/pull/1884/commits/a99ad684602e32903affea5b62ad33e91be6a895. Writing now to a file in `/tmp` so in case the system can decide to purge/clean up it.

> `[Step Debug] Could not connect to debugging client. Tried: host.docker.internal:9003 (through xdebug.client_host/xdebug.client_port).`
This is normal. xdebug is automatically always enabled and will try to reach your IDE or debugging server in general. If you let PHPStorm listen on the default 9003 port that message will go away